src/bin/winkernel/msquicpcw.c

```diff
@@ -0,0 +1,190 @@
+#include <wdm.h>
+#include <ip2string.h>
+#include "msquic.h"
+#include "quic_platform_winkernel.h"
+
+#pragma code_seg(push, "PAGE")
+
+EXTERN_C DECLSPEC_SELECTANY PPCW_REGISTRATION MsQuicPcwGlobal = NULL;
+
+EXTERN_C FORCEINLINE VOID
```

Comment:
Remove the `FORCEINLINE`. Replace Windows types with C types where possible (`VOID` -> `void`). Same below.